Alexintosh/PIE-landing
# Footer links and style

##### Description
We want to have a footer in order to share links to docs, social etc.
Some of the links (like twitter) are not ready yet.

##### Checklist


- [ x ] Changes don't break existing behavior
- [ x ] Tests coverage hasn't decreased

##### Testing
- Verify that the footer looks correctly on every page
- Verify that links work
- Verify that Mixpanel is tracking links

##### Refers/Fixes
Issue #112
